templates/style/base.scss

```diff
@@ -444,87 +444,119 @@ div.cratesfyi-package-container {
     border-bottom: 1px solid $color-border;
     margin-bottom: 20px;
 
-    h1 {
-        margin: 0;
-        padding: 15px 0 0 14px;
+    .container {
+        display: flex;
+        align-items: center;
```

Comment:
I don't feel comfortable reviewing all this CSS, but since you and pietro worked it out I trust your judgement. The end result looks good :)